### ISSUE TITLE
Suddenly getting an error when trying to use Save Analyzer

### ISSUE THREAD

**Issue by Merkava2k15** [AndrewSav/Remnant2SaveAnalyzer#24]
So save analyzer was working perfectly for me (great work btw) when it just suddenly "broke" the other day and just kept giving me the following error when trying to view the world analyzer:

```
03 Jun 2024 16:07:40 ERR [Notifications] System.NullReferenceException: Object reference not set to an instance of an object.
   at lib.remnant2.analyzer.Model.LootItemContext.GetActor(String name)
   at lib.remnant2.analyzer.CustomScripts.NecklaceOfFlowingLife(LootItemContext lic)
   at lib.remnant2.analyzer.CustomScripts.<>c.<.cctor>b__26_3(LootItemContext lic)
   at lib.remnant2.analyzer.Analyzer.<>c__DisplayClass14_0.<FillLootGroups>g__ProcessPrerequisitesAndScripts|1(Zone zone, Location location, LootGroup lootGroup, LootItem lootItem)
   at lib.remnant2.analyzer.Analyzer.FillLootGroups(RolledWorld world)
   at lib.remnant2.analyzer.Analyzer.Analyze(String folderPath, Dataset oldDataset)
   at Remnant2SaveAnalyzer.RemnantSave.UpdateCharacters() in Remnant2SaveAnalyzer\RemnantSave.cs:line 105
```

Not sure what caused it but it's remained broken ever since, Save Guardian still works but for some reason Save Analyzer be ded for me now.

**Comment by AndrewSav** [comment]
@Merkava2k15 attach all the *.sav please: the profile.sav and all the save_x.sav's. Thanks!

**Comment by AndrewSav** [comment]
Actually never, mind, it should be fixed in 0.0.17.
